### PR TITLE
Wishlist managment

### PR DESCRIPTION
**Pull request by Sharmaryan**
**functionality**

- add to wishlist
- remove from wishlist
- add to wishlist button changes to move to wishlist on clicking

**fixes**
- added className , removed class
- netlify error fixed

### CODE REVIEW

File: src/components/WishlistProduct/WishlistProduct.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          <div class="card card-dismiss">
+            <div class="card-dismiss-btn">
+              <i
+                class="fa fa-times"
+                onClick={() => removeFromWishlist(items)}
+              ></i>
+            </div>
+            <img src={image} alt="" class="card-logo" />
+            <p class="card-title">{name}</p>
+            <div class="card-price">{price}</div>
```
Comment (agrawalshanu708) [inline src/components/WishlistProduct/WishlistProduct.jsx:22]: either remove alt or somethng there

**Review by agrawalshanu708 — COMMENTED** [review]
looks good to merge

File: src/components/WishlistProduct/WishlistProduct.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        const { image, name, price } = items;
+        return (
+          <div class="card card-dismiss">
+            <div class="card-dismiss-btn">
+              <i
+                class="fa fa-times"
+                onClick={() => removeFromWishlist(items)}
+              ></i>
+            </div>
+            <img src={image} alt="" class="card-logo" />
```
Comment (zabihshaik) [inline src/components/WishlistProduct/WishlistProduct.jsx]: Add some meaningful description in alt for image

**Review by zabihshaik — APPROVED** [review]
Everything looks good. Just a suggestion to add alt text in image.
UI looks good.

File: src/components/WishlistProduct/WishlistProduct.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useWishlist } from "context/wishlist-context";
+import "./WishlistProduct.css";
+export const WishlistProduct = () => {
+  const { wishlistItems, removeFromWishlist, moveToCart, errorMsg } =
+    useWishlist();
+  return (
+    <div className="wishlist-items">
+      {errorMsg && <h1>item is already in the cart</h1>}
+      {wishlistItems.map((items) => {
```
Comment (nyyshaaa) [inline src/components/WishlistProduct/WishlistProduct.jsx]: you can use item here instead of items as it is singular

**Review by nyyshaaa — COMMENTED** [review]
Not able to add items to wishlist , also login page not coming if i click on add to wishlist .
Destructuring is good.

**Review by nyyshaaa — APPROVED** [review]
Sorry It's working fine,you can add login screen to come when not logged in and clicking on add to wishlist ,i guess you gonna add it later,earlier i didn't try to login from login page .

**Comment by Sharmaryan** [comment]
> Sorry It's working fine,you can add login screen to come when not logged in and clicking on add to wishlist ,i guess you gonna add it later,earlier i didn't try to login from login page .

yes, because I didn't much about it and deadline is already there, so I will do it after that

**Pull request merged**